README.md

```diff
@@ -48,7 +49,8 @@ Example
 
     .:15353 {
         docker unix:///var/run/docker.sock {
-            domain docker.local
+            domain docker.loc
```

Comment:
Is that a typo? `.loc` vs `.local`

Reply:
Probably cause of https://github.com/laravel/valet/issues/566#issuecomment-380421645